hbjs.js

```diff
@@ -238,14 +238,15 @@ function hbjs(instance) {
     };
   }
 
-  var utf8Encoder = new TextEncoder("utf8");
-  function createCString(text) {
-    var bytes = utf8Encoder.encode(text);
-    var ptr = exports.malloc(bytes.byteLength);
-    heapu8.set(bytes, ptr);
+  function createAsciiString(text) {
+    var ptr = exports.malloc(text.length + 1);
+    for (let i = 0; i < text.length; ++i) {
+      heapu8[ptr + i] = text.charCodeAt(i);
```

Comment:
This this takes user input, I’d fo some error checking here that the code point is indeed in the ASCII range.